src/admin/UserList.tsx

```diff
@@ -45,6 +47,39 @@ const UserList = () => {
   const editUserRef = useRef({ submit: () => Promise.resolve() });
   const createUserRef = useRef({ submit: () => Promise.resolve() });
 
+  const columns = useMemo<Column<User>[]>(
```

Comment:
Instead of memo'ing this, we can just pull it out into the top-level, above the component.

Reply:
Fair cop. 8f2d1c4e